packages/dds/matrix/src/matrix.ts

```diff
@@ -128,18 +134,20 @@ export class SharedMatrix<T extends Serializable = Serializable>
     /**
      * {@inheritDoc @fluidframework/datastore-definitions#IChannelFactory.create}
      */
-    public static create<T extends Serializable = Serializable>(runtime: IFluidDataStoreRuntime, id?: string) {
+    public static create<T>(runtime: IFluidDataStoreRuntime, id?: string) {
         return runtime.createChannel(id, SharedMatrixFactory.Type) as SharedMatrix<T>;
     }
 
     // #region IMatrixProducer
 
-    openMatrix(consumer: IMatrixConsumer<T | undefined | null>): IMatrixReader<T | undefined | null> {
+    openMatrix(
+        consumer: IMatrixConsumer<MatrixItem<T>>,
```

Comment:
Is it intentional to start excluding `null`?

Reply:
Same for signatures below too.

Reply:
It is.  SharedMatrix reserves 'null' as a sentinel value for internal use.  Thx.